format-all.el

```diff
@@ -853,6 +855,12 @@ Consult the existing formatters for examples of BODY."
   (:languages "_Snakemake")
   (:format (format-all--buffer-easy executable "-")))
 
+(define-format-all-formatter zig
+  (:executable "zig")
+  (:install)
+  (:languages "Zig")
+  (:format (format-all--buffer-easy executable "fmt" "--stdin")))
+
```

Comment:
Please keep the formatter definitions in alphabetical order, i.e. move `(define-format-all-formatter zig ...)` so it's the last definition.